spec/rubocop/cop/rspec/rails/travel_around_spec.rb

```diff
@@ -0,0 +1,144 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::Rails::TravelAround do
+  context 'with `freeze_time` in `before`' do
+    it 'registers no offense' do
+      expect_no_offenses(<<~RUBY)
+        before { freeze_time }
+      RUBY
+    end
+  end
+
+  context 'with `freeze_time` in `around(:all)`' do
+    it 'registers no offense' do
+      expect_no_offenses(<<~RUBY)
+        around(:all) do |example|
+          freeze_time do
+            example.run
+          end
+        end
+      RUBY
+    end
+  end
+
+  context 'with `freeze_time` in `around(:suite)`' do
+    it 'registers no offense' do
+      expect_no_offenses(<<~RUBY)
+        around(:suite) do |example|
+          freeze_time do
+            example.run
+          end
+        end
+      RUBY
+    end
+  end
+
+  context 'with `freeze_time` in `around`' do
+    it 'registers offense' do
+      expect_offense(<<~RUBY)
+        around do |example|
+          freeze_time do
+          ^^^^^^^^^^^^^^ Prefer to travel in `before` rather than `around`.
+            example.run
+          end
+        end
+      RUBY
+
+      expect_correction(<<~RUBY)
+        before { freeze_time }
+
+        around do |example|
+          example.run
+        end
+      RUBY
+    end
+  end
+
+  context 'with `freeze_time` in `around(:each)`' do
+    it 'registers offense' do
+      expect_offense(<<~RUBY)
+        around(:each) do |example|
+          freeze_time do
+          ^^^^^^^^^^^^^^ Prefer to travel in `before` rather than `around`.
+            example.run
+          end
+        end
+      RUBY
+
+      expect_correction(<<~RUBY)
+        before { freeze_time }
+
+        around(:each) do |example|
+          example.run
+        end
+      RUBY
+    end
+  end
+
+  context 'with `freeze_time` and another node in `around`' do
+    it 'registers offense' do
+      expect_offense(<<~RUBY)
+        around do |example|
```

Comment:
What if we have
```
around do |example|
  freeze_time do
     do_some_preparation
     example.run
  end
end
```
So the do_some_preparation also relies on the time freeze?

Reply:
The above example is ignored, since it is only violated if there is only `example.run` in `freeze_time { ... }` block 👌

Reply:
@r7kamura  Can you please add a test case for this 👍  I have no other remarks and would be happy to merge. Sorry for the great delay in reviewing this

Reply:
Thanks, I've added that test case 👌